bin/setup-production

```diff
@@ -20,20 +24,17 @@ end
 APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
 
 chdir APP_ROOT do
-  # get env vars for DATABASE and DATAROOT
-  Dotenv.load(".env.local", ".env.production", ".env")
-
-  `mkdir -p #{ENV["OOD_DATAROOT"]}` if ENV["OOD_DATAROOT"]
+  `mkdir -p #{AppConfig.dataroot}` if AppConfig.dataroot
```

Comment:
`AppConfig.dataroot` will always exist, so the trailing `if ...` is probably not necessary anymore.

Reply:
Fixed.